index.html

```diff
@@ -7,27 +7,58 @@
     <title>Project One</title>
     <meta name='viewport' content='width=device-width, initial-scale=1'>
 
+    <!-- Purecss Library -->
+    <link rel="stylesheet" href="https://unpkg.com/purecss@2.0.3/build/pure-min.css" integrity="sha384-cg6SkqEOCV1NbJoCu11+bm0NvBRc8IYLRGXkmNrqUBfTjmMYwNKPWBTIKyw9mHNJ" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://unpkg.com/purecss@2.0.3/build/grids-responsive-min.css">
 
-    <!-- Team Created CSS Stylesheet -->
-    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
-
+    <!-- Team CSS file -->
+    <link rel='stylesheet' type='text/css' media='screen' href='projectOne.css'>
```

Comment:
projectOne.css just needs to be changed to style.css, since that's the name of the css file
Otherwise, all of this looks great!